README2.md

```
@@ -0,0 +1,30 @@
+# How to Run the Application
+1. Start MySQL in Docker
+   docker-compose up -d
+
+2. Run Spring Boot
+   mvn spring-boot:run
+
+3. Open the HTML page
+   http://localhost:8080/quotes
+
+4. Test the REST API
+   GET http://localhost:8080/api/quotes
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)